js/interface.js

```diff
@@ -421,6 +421,11 @@ function submissionBuild(appSubmission, origin) {
   }, function(err) {
     $('.button-' + origin + '-request').html('Request App <i class="fa fa-paper-plane"></i>');
     $('.button-' + origin + '-request').prop('disabled', false);
+
+    if (Fliplet.Error.isHandled(err)) {
```

Comment:
@hcarneiro there is no track of this change from https://github.com/Fliplet/fliplet-widget-google-submission/pull/76 in the deployment sheet.

cc @tonytlwu

Reply:
@squallstar this PR added that: https://github.com/Fliplet/fliplet-widget-google-submission/pull/76
That is marked as released in the sheet.